metadata.yaml

```diff
@@ -23,6 +26,9 @@ provides:
   microk8s-provides:
```

Comment:
I wonder if we can make the relation setup a bit nicer: `juju relate microk8s:microk8s-provides microk8s-worker:microk8s`. How about `juju relate microk8s-control-plane:workers microk8s-workers:control-plane`? It may seem awkward in our code but the user can read it as "connect the control-plane workers to the workers control-plane".